client/src/main/proto/spine/client/subscription.proto

```diff
@@ -86,12 +86,35 @@ message SubscriptionUpdate {
     core.Response response = 2 [(required) = true];
 
     // Reserved for more subscription update attributes.
-    reserved 3 to 9;
+    reserved 3 to 8;
 
-    // Entity updates packed as Any.
+    // The updates of the Entity states.
     //
-    // Each of the update messages is affected by the field mask set for the current subscription.
-    repeated google.protobuf.Any updates = 10;
+    // Each update is a pair of an Entity ID and state.
+    //
+    // Each of the update state messages is affected by the field mask set for the current
+    // subscription.
+    //
+    repeated EntityStateUpdate entity_state_updates = 9 [(valid) = true];
+
+    // Deprecated field for the Entity state updates.
+    //
+    // Use `entity_state_updates` instead.
+    //
+    repeated google.protobuf.Any OBSOLETE_updates = 10 [deprecated = true];
```

Comment:
Let's have this field `reserved` as discussed. 
Also, please document it properly.